### PR TITLE
Update Communications Handbook for adusting timeline

### PR DESCRIPTION
**Pull request by mkorbi**
In https://github.com/kubernetes/sig-release/issues/1024 we discussed that the SIGs do not have enough time to prepare additional blog posts. A reason for this is the given timeline for the comms team in collaboration with the SIGs is to tight.

/kind documentation
/sig release
/assign @onlydole
/hold

### CODE REVIEW

**Comment by onlydole** [comment]
Thank you for updating this, @mkorbi! I talked to @justaugustus on this topic a bit today and we have some ideas on how to ensure a bit better process and handoff between the Enhancements and Communications team during a release. Let me think on this over the weekend and I can return to it, this coming week.

**Comment by mkorbi** [comment]
/assign @palnabarun 
We rework the timeline and added stronger sync between Enhancement and Comms, therefore I added an additional row to the enhancement timeline. Happy about every feedback :)

File: release-team/role-handbooks/enhancements/README.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ It is important that this process be followed and documentation remain up-to-dat
 | 1 | Collection | Start reminding Issue owners that KEPs are required for each enhancement and that KEPs must be in an implementable state by Enhancement Freeze. |
 | 2-3 | Collection Monitoring and Triage | Stay on top of comments in issues when owners respond. Apply correct labels, milestone information, update their status in the sheet if necessary. |
 | 2-3 | Collection Monitoring and Triage | Mark features as `At Risk` if there is no communication, active PRs on the issues, or it is missing other requirements coming into Enhancement Freeze. |
+| 2-3 | Communication Alignment| Start syncing with Communications Team on giving an induction what's coming up for the release.  |
```
Comment (palnabarun) [inline release-team/role-handbooks/enhancements/README.md:104]: _nit_
```suggestion
| 2-3 | Communication Alignment | Start syncing with Communications Team on giving an induction what's coming up for the release. |
```

**Comment by mkorbi** [comment]
good to go for 1.19? @onlydole @justaugustus

**Comment by onlydole** [comment]
I like it, @mkorbi!

/approve

**Pull request merged**